src/sentry/models/authprovider.py

```diff
@@ -10,6 +12,14 @@
     sane_repr,
 )
 
+logger = logging.getLogger("sentry.authprovider")
+
+SCIM_INTERNAL_INTEGRATION_OVERVIEW = (
```

Comment:
someone could go in and delete this token and it would cause SCIM to stop working -- there was discussion when this feature was added to have a notion of "undeletable" internal integrations, could be worth adding that in now. (need to guard against this case elsewhere if we don't)